.github/ISSUE_TEMPLATE/4.docs_request.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Docs Request for an Update or Improvement'
+description: A task to update or improve Next.js documentation
+title: ''
```

Comment:
Why is `title` empty?

Reply:
Oh i need to fix that. The idea was to not prefix the title as it could be anything, but this will fail as yaml needs a string value here. Will add a "Docs: " prefix